ztp/source-crs/StorageLVMOSubscription.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: odf-lvm-operator
+  namespace: openshift-storage
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+spec:
+  channel: "stable-4.11"
```

Comment:
Is there a non-versioned "stable" channel? Other operators are not pinned to a specific channel

Reply:
Currently only stable-4.11 and stable-4.10 are available for 4.11 and stable-4.11 for OCP 4.12:

```
oc-mirror list operators --catalog=registry.redhat.io/redhat/redhat-operator-index:v4.11 --package=odf-lvm-operator
NAME              DISPLAY NAME      DEFAULT CHANNEL
odf-lvm-operator  ODF LVM Operator  stable-4.11

PACKAGE           CHANNEL      HEAD
odf-lvm-operator  stable-4.10  odf-lvm-operator.v4.10.8
odf-lvm-operator  stable-4.11  odf-lvm-operator.v4.11.3

# oc-mirror list operators --catalog=registry.redhat.io/redhat/redhat-operator-index:v4.12 --package=odf-lvm-operator
NAME              DISPLAY NAME      DEFAULT CHANNEL
odf-lvm-operator  ODF LVM Operator  stable-4.11

PACKAGE           CHANNEL      HEAD
odf-lvm-operator  stable-4.11  odf-lvm-operator.v4.11.3
```